### PR TITLE
style: 2021-09-20 linter warnings batch 20 / 26 #1369

### PR DESCRIPTION
**Pull request by micoferdinand98**
style: 2021-09-20 linter warnings batch 20 / 26 

 changes made in assigned lines 191-200

### CODE REVIEW

**Comment by micoferdinand98** [comment]
image


added DCO signed off on my commits

**Comment by petermetz** [comment]
> image
> 
> added DCO signed off on my commits

@micoferdinand98 We are almost there. Now you just need to make sure that there's only a single commit, not 2 or more. (squash with an interactive rebase is what I usually do - let me know if you need more assistance with the git mechanics)

File: packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts [hunk 1]
```diff
@@ -359,19 +359,19 @@ test(testCase, async (t: Test) => {
         },
       };
       await apiClient.runTransactionV1(
-        (parameters as any) as RunTransactionV1Request,
+        (parameters as unknown) as RunTransactionV1Request,
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts:362]: @micoferdinand98 IIRC this cast can be eliminated entirely. Could you please double check?

File: packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts [hunk 2]
```diff
@@ -359,19 +359,19 @@ test(testCase, async (t: Test) => {
...
         400,
         `Endpoint ${fRun} without required consistencyStrategy: response.status === 400 OK`,
       );
-      const fields = e.response.data.map((param: any) =>
+      const fields = e.response.data.map((param: { path: string }) =>
         param.path.replace(".body.", ""),
       );
       t2.ok(
-        fields.includes("consistencyStrategy"),
+        fields?.includes("consistencyStrategy"),
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts:374]: @micoferdinand98 I think the question mark is not needed anymore, could you please double check?

File: packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts [hunk 3]
```diff
@@ -401,19 +401,19 @@ test(testCase, async (t: Test) => {
         fake: 4,
       };
       await apiClient.runTransactionV1(
-        (parameters as any) as RunTransactionV1Request,
+        (parameters as unknown) as RunTransactionV1Request,
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts:404]: @micoferdinand98 IIRC this cast can be eliminated entirely. Could you please double check?

File: packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts [hunk 4]
```diff
@@ -401,19 +401,19 @@ test(testCase, async (t: Test) => {
...
         400,
         `Endpoint ${fRun} with fake=4: response.status === 400 OK`,
       );
-      const fields = e.response.data.map((param: any) =>
+      const fields = e.response.data.map((param: { path: string }) =>
         param.path.replace(".body.", ""),
       );
       t2.ok(
-        fields.includes("fake"),
+        fields?.includes("fake"),
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts:416]: @micoferdinand98 I think the question mark is not needed anymore, could you please double check?

File: packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts [hunk 5]
```diff
@@ -296,19 +296,19 @@ test(testCase, async (t: Test) => {
...
         400,
         `Endpoint ${fInvoke} with fake=4: response.status === 400 OK`,
       );
-      const fields = e.response.data.map((param: any) =>
+      const fields = e.response.data.map((param: { path: string }) =>
         param.path.replace(".body.", ""),
       );
       t2.ok(
-        fields.includes("fake"),
+        fields?.includes("fake"),
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts:311]: @micoferdinand98 I think the question mark is not needed anymore, could you please double check?

File: packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts [hunk 6]
```diff
@@ -296,19 +296,19 @@ test(testCase, async (t: Test) => {
         fake: 4,
       };
       await apiClient.invokeContractV1(
-        (parameters as any) as InvokeContractV1Request,
+        (parameters as unknown) as InvokeContractV1Request,
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts:299]: @micoferdinand98 IIRC this cast can be eliminated entirely. Could you please double check?

File: packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts [hunk 7]
```diff
@@ -263,15 +263,15 @@ test(testCase, async (t: Test) => {
...
         400,
         `Endpoint ${fInvoke} without required contractName: response.status === 400 OK`,
       );
-      const fields = e.response.data.map((param: any) =>
+      const fields = e.response.data.map((param: { path: string }) =>
         param.path.replace(".body.", ""),
       );
       t2.ok(
-        fields.includes("contractName"),
+        fields?.includes("contractName"),
```
Comment (petermetz) [inline packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts:274]: @micoferdinand98 I think the question mark is not needed anymore, could you please double check?

**Review by petermetz — CHANGES_REQUESTED** [review]

**Comment by petermetz** [comment]
@micoferdinand98 One more thing: I recommend using the issue title as the commit message and also as the PR title.
Saying `fixed batch` is easy to confuse with any other PR that is doing the same thing.
Note: Please do not close down the PR, these changes can be made on the current one, in-place, and then the review comment history does not get erased either.

File: packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts (hunk 2 above)
Comment (micoferdinand98) [reply packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts:374]: yes sir its no longer needed, i overlooked this one .

**Comment by micoferdinand98** [comment]
> @micoferdinand98 One more thing: I recommend using the issue title as the commit message and also as the PR title. Saying `fixed batch` is easy to confuse with any other PR that is doing the same thing. Note: Please do not close down the PR, these changes can be made on the current one, in-place, and then the review comment history does not get erased either.

Copy that, from now on will make sure that I conform my commits & PRs

**Comment by micoferdinand98** [comment]
> > image
> > added DCO signed off on my commits
> 
> @micoferdinand98 We are almost there. Now you just need to make sure that there's only a single commit, not 2 or more. (squash with an interactive rebase is what I usually do - let me know if you need more assistance with the git mechanics)

Sir Peter I've done the interactive rebase on my branch and applied my changes too.

**Review by petermetz — CHANGES_REQUESTED** [review]
@micoferdinand98 Unfortunately it appears you did not squash the commits into one though:

Screenshot from 2022-01-26 15-51-20

**Comment by micoferdinand98** [comment]
> @micoferdinand98 Unfortunately it appears you did not squash the commits into one though:
> 
> Screenshot from 2022-01-26 15-51-20

I will do the Rebasing again, It must have overlooked last time I did the Interactive Rebasing.

**Review by petermetz — APPROVED** [review]
@micoferdinand98 Thank you, LGTM

**Comment by micoferdinand98** [comment]
> @micoferdinand98 Thank you, LGTM

Thank you Peter for the knowledge transition, I'll be able to successfully do the Interactive Rebase on my own when I've to squash a commit

**Review by izuru0 — APPROVED** [review]
LGTM

**Pull request merged**